src/main/java/com/plugin/frontend/actions/FormatFileAction.java

```diff
@@ -1,29 +1,30 @@
 package com.plugin.frontend.actions;
 
-import com.google.googlejavaformat.java.FormatterException;
 import com.intellij.openapi.actionSystem.AnAction;
 import com.intellij.openapi.actionSystem.AnActionEvent;
 import com.intellij.openapi.actionSystem.LangDataKeys;
 import com.intellij.openapi.fileEditor.FileDocumentManager;
 import com.intellij.openapi.vfs.VirtualFile;
-import com.intellij.util.IconUtil;
 import com.plugin.engine.JavaFormatter;
 import org.jetbrains.annotations.NotNull;
 
 import java.io.File;
-import java.io.IOException;
+import java.util.Objects;
 
 public class FormatFileAction extends AnAction {
     @Override
     public void actionPerformed(@NotNull AnActionEvent e) {
         JavaFormatter test = new JavaFormatter();
-        VirtualFile file = FileDocumentManager.getInstance().getFile(e.getData(LangDataKeys.EDITOR).getDocument());
+        VirtualFile file = FileDocumentManager.getInstance().getFile(Objects.requireNonNull(e.getData(LangDataKeys.EDITOR)).getDocument());
+
+        if (file == null) {
+            return;
+        }
+
         try {
             test.formatFile(new File(file.getPath()));
-        } catch (FormatterException formatterException) {
-            formatterException.printStackTrace();
-        } catch (IOException ioException) {
-            ioException.printStackTrace();
+        } catch (Exception exception) {
```

Comment:
You should use specific exception. Using an exception class in a try-catch block is a bad idea.